config/mainmenu.json

```diff
@@ -1,22 +1,60 @@
 {
 	//images used in game selection screen
-	"game-select" : ["gamselb0", "gamselb1"],
-
-	//Loading screen background and progress bar
+	"scenario-selection" :
+	{
+		"background" : ["gamselb0", "gamselb1"],
+		
+		//"logo": {
+		//	"name": ["logo/logo0", "logo/logo1", "logo/logo2"], 
+		//	"x": 20, "y": 20
+		//},
+		//"sublogo": {
+		//	"name": ["sublogo/sublogo0", "sublogo/sublogo1", "sublogo/sublogo2"], 
+		//	"x": 20, "y": 120
+		//}
```

Comment:
Perhaps format like this would be more suiltable? Why VCMI should care about what these images represent? It could be logo or could be something else:
```json
    "images": [
        {
            "x": 20, 
            "y": 20,
            "name": ["logo/logo0", "logo/logo1", "logo/logo2"] // randomly selected
        },
        {
            "x": 20, 
            "y": 120,
            "name": ["sublogo/sublogo0", "sublogo/sublogo1", "sublogo/sublogo2"] // randomly selected
        }
```

Reply:
Looks better, but I have no idea how to do that in code as their variables are directly in headers.

Reply:
Something like this is my goal. Logo = Main Heroes 3 text and SubLogo expansion like text.
![image](https://github.com/user-attachments/assets/6e59508a-7453-4152-8396-e10ef4cefa47)
![image](https://github.com/user-attachments/assets/bd561221-29e4-4d97-9b50-870d5a155678)
![image](https://github.com/user-attachments/assets/c0b0497c-4037-4129-8189-16abeac28759)